### PR TITLE
[WIP] Improve scripts

### PR DESCRIPTION
**Pull request by danielaRiesgo**
Separating download from actions.
Adding parameters to all general and cnn scripts instead of manually changing things.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,15 +18,21 @@ python ./mnist_tutorial/mnist_deep.py
 ## Grabar video
 
 ### Grabar como secuencia de imágenes
-1. Modificar en el archivo `capture_frames.py` la carpeta en donde querés que se te guarden las fotos.
-La carpeta **debe existir**.
-2. Correr `capture_frames.py`.
+1. Correr `python capture_frames.py --video_dir ./<path_a_carpeta>`.
 Las imágenes van a tener el formato `YYYY-MM-dd hh:mm:ss.ms`
```
Comment (fmesteban) [inline README.md]: Update this

File: README.md [hunk 2]
```diff
@@ -36,7 +44,12 @@ Tenés que crear dentro de `cnn/photos` una carpeta por cada categoría que quie
 
 2. Corré el comando:
 ```
-python ./cnn/retrain.py --bottleneck_dir=./cnn/bottleneck --model_dir=./cnn/inception --output_graph=./cnn/retrained_graph.pb --output_labels=./cnn/retrained_labels.txt --image_dir ./cnn/photos
+python ./cnn/retrain.py
+	--bottleneck_dir=./cnn/bottleneck
+	--model_dir=./cnn/inception
+	--output_graph=./cnn/retrained_graph.pb
+	--output_labels=./cnn/retrained_labels.txt
+	--image_dir ./cnn/photos
```
Comment (fmesteban) [inline README.md]: I've seen that the = are replaceable with spaces. I'd use only one of the two just to be consistent.

File: capture_frames.py [hunk 3]
```diff
@@ -1,13 +1,16 @@
 from __future__ import print_function
-import numpy as np
+import os
+import argparse
 import datetime
+import numpy as np
```
Comment (fmesteban) [inline capture_frames.py]: Alphabetical order pls.

You can check the retrain file to see how that files does it which I think is quite nice.

File: capture_frames.py [hunk 4]
```diff
@@ -19,32 +22,49 @@ def capture_images(save_folder):
 	print("La resolucion es de {}x{}".format(camera.get(cv2.CAP_PROP_FRAME_WIDTH), camera.get(cv2.CAP_PROP_FRAME_HEIGHT)))
 	print("Se pudo cambiar el frame count a 10: {}".format(camera.set(cv2.CAP_PROP_FPS, 10)))
 	print("El frame count es de {}".format(camera.get(cv2.CAP_PROP_FPS)))
-	
+
 	count = 0
-	
+
 	while(True):
 		# Capture frame-by-frame
```
Comment (fmesteban) [inline capture_frames.py]: so much space

File: capture_frames.py [hunk 5]
```diff
@@ -19,32 +22,49 @@ def capture_images(save_folder):
...
-			timestamp = datetime.datetime.now()
+
+			# Windows doesn't accept ":" in file name
+			# and cv2.imwrite will silently fail
+			timestamp = str(datetime.datetime.now()).replace(":", "-")
+
 			img_path = os.path.join(save_folder, "{}.jpg".format(timestamp))
 			cv2.imwrite(img_path, frame)
-
+			print("pathhh: ", img_path)
```
Comment (fmesteban) [inline capture_frames.py]: hhhh

File: cnn/real_time.py [hunk 6]
```diff
@@ -63,4 +64,23 @@ def capture_images(labels):
...
+      type=str,
+      default='/tmp/output_labels.txt',
+      help="""\
+      Where to find the output_labels of the retrain.py script.\
+      """
+  )
+  parser.add_argument(
+      '--retrained_graph',
+      type=str,
+      default='/tmp/output_graph.pb',
```
Comment (fmesteban) [inline cnn/real_time.py]: This'll work in Windows?

File: cnn/classify_files.py [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+		_ = tf.import_graph_def(graph_def, name='')
+
+	with tf.Session() as sess:
+		softmax_tensor = sess.graph.get_tensor_by_name('final_result:0')
+
+		for base_dir, dirs, files in os.walk(source):
+			for file in files:
+				frame = cv2.imread(os.path.join(base_dir, file))
+
+
```
Comment (fmesteban) [inline cnn/classify_files.py]: Extra space

File: cnn/real_time.py (hunk 6 above)
Comment (danielaRiesgo) [reply cnn/real_time.py]: Yes, it does. Thank God

**Review by fmesteban — DISMISSED** [review]

**Review by fmesteban — APPROVED** [review]

**Pull request merged**